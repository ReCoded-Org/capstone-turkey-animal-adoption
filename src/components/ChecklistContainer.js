import React from "react";
import Checklist from "./Checklist";

const ChecklistContainer = ({data}) => {
    console.log(data)
    return(
        <div>
            {data.map((info)=><Checklist text={info.text} title={info.title} id={info.id}/>)}
        </div>
    )
}

export default ChecklistContainer;