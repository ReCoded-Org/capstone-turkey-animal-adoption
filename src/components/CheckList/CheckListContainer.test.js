import React from "react";
import renderer from "react-test-renderer";
import ChecklistContainer from "./CheckListContainer";

const data = [
  {
    text: "Go to Checklist",
    title: "Cat Adoption Checklist",
    link: "denemeLink",
    id: "id1",
  },
  {
    text: "Go to Checklist",
    title: "Cat Adoption Checklist",
    link: "denemeLink",
    id: "id2",
  },
  {
    text: "Go to Checklist",
    title: "Cat Adoption Checklist",
    link: "denemeLink",
    id: "id3",
  },
];

it("checklist container renders correctly", () => {
  const tree = renderer.create(<ChecklistContainer data={data} />).toJSON();
  expect(tree).toMatchSnapshot();
});
