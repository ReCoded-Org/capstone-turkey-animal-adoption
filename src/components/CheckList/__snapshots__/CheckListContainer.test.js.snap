// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checklist container renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="py-5 row"
  >
    <div
      className="col-lg-4"
    >
      <div
        className="checkList"
      >
        <div
          className="checkListLogo"
        >
          <img
            className="catLogo"
            src="catLogo.png"
          />
        </div>
        <h2>
          Cat Adoption Checklist
        </h2>
        <p
          className="checkListText"
        >
          Go to Checklist
        </p>
        <a
          href="denemeLink"
        >
          <button
            className="checkListButton"
          >
            Read More
          </button>
        </a>
      </div>
    </div>
    <div
      className="col-lg-4"
    >
      <div
        className="checkList"
      >
        <div
          className="checkListLogo"
        >
          <img
            className="catLogo"
            src="catLogo.png"
          />
        </div>
        <h2>
          Cat Adoption Checklist
        </h2>
        <p
          className="checkListText"
        >
          Go to Checklist
        </p>
        <a
          href="denemeLink"
        >
          <button
            className="checkListButton"
          >
            Read More
          </button>
        </a>
      </div>
    </div>
    <div
      className="col-lg-4"
    >
      <div
        className="checkList"
      >
        <div
          className="checkListLogo"
        >
          <img
            className="catLogo"
            src="catLogo.png"
          />
        </div>
        <h2>
          Cat Adoption Checklist
        </h2>
        <p
          className="checkListText"
        >
          Go to Checklist
        </p>
        <a
          href="denemeLink"
        >
          <button
            className="checkListButton"
          >
            Read More
          </button>
        </a>
      </div>
    </div>
  </div>
</div>
`;
